chain/chunks/src/lib.rs

```diff
@@ -982,17 +978,17 @@ impl ShardsManager {
     pub fn reintroduce_transactions(
         &mut self,
         shard_id: ShardId,
-        transactions: &Vec<SignedTransaction>,
+        transactions: &[SignedTransaction],
     ) {
-        self.pool_for_shard(shard_id).reintroduce_transactions(transactions.clone());
+        self.pool_for_shard(shard_id).reintroduce_transactions(transactions.to_vec());
     }
```

Comment:
I've stared at this one for some time a while back. Ideally, this would take an owned `Vec<>`, but, IIRC, for some reason this wasn't an obvious change either.